packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -471,6 +372,25 @@ test.serial('writes GTM account addresses to vstorage', async t => {
   await documentStorageSchema(t, storage, doc);
 });
 
+test.serial('writes pool metrics to vstorage', async t => {
+  const { storage } = t.context;
+  const doc = {
+    node: 'fastUsdc.poolMetrics',
+    owner: 'FastUSC LiquidityPool exo',
+    showValue: defaultSerializer.parse,
+  };
+  await documentStorageSchema(t, storage, doc);
+});
+
+test.serial('upgrade 2; update Settler reference', async t => {
```

Comment:
I think we'll regret enumerating upgrades. there's also `incarnationNumber`, which increments on non-upgrading restart.

II find it unfortunate that we have the concept of a "null upgrade" but we do.

Consider `deploy HEAD; update Settler reference`.